### ISSUE TITLE
Incorrect tip for the Sort button on Tools > Add-ons page

### ISSUE THREAD

**Issue by ccfs** [tablacus/TablacusExplorer#654]
@tablacus, please help to solve this problem. Thanks.
![opt sort](https://user-images.githubusercontent.com/34089121/213840073-3ae3264d-2fb7-49b3-a578-3308b5d44a87.png)

**Comment by tablacus** [comment]
I see, but  there is no issue in my environment.
Does "wmproc.dll" not exist in "C:\Windows\System32" in your environment?

![image](https://user-images.githubusercontent.com/5156977/213866736-32831184-7031-4ab1-91fb-bab699be9a33.png)

![image](https://user-images.githubusercontent.com/5156977/213866768-5bd39092-7a73-4fa4-8f04-bde51ae6dfaa.png)

![image](https://user-images.githubusercontent.com/5156977/213866802-f45e36b8-b65b-4bca-a0f6-ec8b6bbbac6f.png)

Regards,

**Comment by ccfs** [comment]
No, "wmploc.dll" does not exist in my system because I have uninstalled Windows Media Player, as I uses another media player. TE should not get any caption from another app; otherwise this causes an unnecessary dependency.
